Portal_Perfomance_Employes/Data/AppDbContext.cs

```diff
@@ -230,6 +230,26 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 new TeamMember { Id = 10, EmployeeId = 11, TeamId = 2, FirstName = "Emp11", LastName = "Test11", FromDate = DateTime.Now, IsActive = true },
                 new TeamMember { Id = 11, EmployeeId = 4, TeamId = 2, FirstName = "Emp4", LastName = "Test4", FromDate = DateTime.Now, IsActive = true }
                 );
+            modelBuilder.Entity<EmployeeSkill>().HasData(

```

Comment:
Can you use data seeding from my last PR so we don't have double population of the same IDs or conflicts when merging? Alternatively I can overwrite mine

Reply:
Thanks for the notice, I'll do it in a new commit